README.md

```diff
@@ -14,17 +14,16 @@ impress.js may not help you if you have nothing interesting to say ;)
 HOW TO USE IT
 ---------------
 
-[Use the source](http://github.com/impress/impress.js/blob/master/index.html), Luke ;)
+[Use the source](http://github.com/impress/impress.js/blob/master/index.html), Luke!
 
-Please note that impress.js was created for developers. Some basic knowledge of html, css and
-javascript is a prerequisite to install and use its features.
+Please note that impress.js was created for developers. Some basic knowledge of HTML, CSS and
```

Comment:
The PR is about "Minor grammatical" tweaks, so changing the way the README is structured goes against [the concern of this PR](https://medium.com/@fagnerbrack/one-pull-request-one-concern-e84a27dfe9f1), right?